staging/src/k8s.io/apiserver/pkg/endpoints/openapi/openapi.go

```diff
@@ -30,10 +30,10 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-	"k8s.io/apiserver/pkg/util/trie"
+	"k8s.io/kube-openapi/pkg/util"
```

Comment:
k8s.io/utils is a new form repo and for this instance I actually think we can move it there. Just for the sake of getting this PR in faster, I suggest we do it in another PR.